activerecord/lib/active_record/railtie.rb

```diff
@@ -88,6 +88,29 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "Check for cache versioning support" do
+      config.after_initialize do |app|
+        ActiveSupport.on_load(:active_record) do
+          if app.config.active_record.cache_versioning && Rails.cache
+            unless Rails.cache.try(:supports_in_cache_versioning?)
```

Comment:
Let's use a class method and normalize naming: `Rails.cache.class.supports_cache_versioning?`

Reply:
I made a comment about the naming in basecamp. I think it's confusing. All caches are versioned already via cache key, the conceptual change is that the versioning is now __in__ the cache which is what i'm trying to convey here with that word. I can standardize on this though.